properties.go

```diff
@@ -562,6 +562,31 @@ func (p *Properties) String() string {
 	}
 	return s
 }
+func (p *Properties) FMap(f func(*Properties) interface{}) interface{} {
```

Comment:
I don't think this belongs here unless you can provide a compelling example of its use. Also, no comment or test.